app/services/session_service.ts

```diff
@@ -0,0 +1,16 @@
+import User from '#models/user'
+
+// interface
+interface SessionServiceStorePayload {
+  email: string
+  password: string
+  first_name: string
+  last_name: string
+}
+
```

Comment:
- Missing Docstring for SessionService class.
- Why is the method named "store" if the function first create a user ? would it not be create_user instead?

Reply:
The method is named "store" because i followed this conventional naming : 
- https://docs.adonisjs.com/guides/controllers#resource-driven-controllers

"For conventional [RESTful](https://en.wikipedia.org/wiki/Representational_state_transfer) applications, a controller should only be designed to manage a single resource. A resource is usually an entity in your application like a User resource or a Post resource."